.github/workflows/lint.yml

```diff
@@ -20,7 +20,7 @@ jobs:
             **/**.go
             go.mod
             go.sum
-      - uses: golangci/golangci-lint-action@v3.1.0
+      - uses: golangci/golangci-lint-action@v2.5.2
```

Comment:
I'm fine with reverting this, but we should probably do it in a separate PR.

Reply:
yeah, I did, I just wanted to see if it would help... this should evaporate in a sec